content/documentation/4-batch-developer-guides/2-batch/3-data-flow-simple-task.md

```diff
@@ -1,32 +1,104 @@
 ---
 path: 'batch-developer-guides/batch/data-flow-simple-task/'
-title: 'Launch a Spring Cloud Task application using Data Flow'
-description: 'Launch a Spring Cloud Task application using Data Flow'
+title: 'Register and Launch a Spring Cloud Task application using Data Flow'
+description: 'Register and Launch a Spring Cloud Task application using Data Flow'
 ---
 
 # Deploy a Spring Cloud Task application using Data Flow
 
-In this section we will show how to register task applications with Data Flow, create a Task DSL, and deploy the task to Cloud Foundry, Kubernetes and your local machine.
+In this section, we will demonstrate how to register a Spring Cloud Task application with Data Flow, create a task definiton, and deplot the task to Cloud Foundry, Kubernetes and your local machine.
 
 ## Development
 
-Explaine what we are doing to do development wise.
+For this guide, we will use the [simple task](batch-developer-guides/batch/simple-task) sample Spring Cloud Task application, called `billsetuptask`.
```

Comment:
The reference to get to simple task is `/documentation/batch-developer-guides/batch/simple-task`